modules/evaluate_condition_abstract.py

```diff
@@ -0,0 +1,21 @@
+"""
+Abstract base class for evaluating drone conditions
+"""
+import abc
+
+
+class EvaluateCondition(abc.ABC):
+    """
+    Wrapper for different classes to check conditions
+    """
```

Comment:
Is `abc` a library that you had to install? If so you need to add it to the `requirements.txt` file.